src/test/scala/firrtlTests/CustomTransformSpec.scala

```diff
@@ -2,10 +2,9 @@
 
 package firrtlTests
 
-import firrtl.ir.Circuit
 import firrtl._
-import firrtl.passes.Pass
-import firrtl.ir._
+import firrtl.ir.{Circuit, _}
```

Comment:
What is the difference between writing `import firrtl.ir._` and `import firrtl.ir.{Circuit, _}`?